app/src/Components/ListTable/ListTableFooter.js

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+    import React from 'react';
```

Comment:
Why indent the import?

Reply:
looks better that way (j/k, wanted to reorder imports to have react as the topmost import in the file)